debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/tdump/zebedee/util/Template.java

```diff
@@ -38,8 +38,8 @@
  * of <a href="http://java.sun.com/j2se/1.4.2/docs/api/javax/imageio/stream/ImageInputStream.html">ImageInputStream</a>
  * so that the template classes can obtain the data for the fields in each structure.
  * <p>
- * The DTD for the XML description of a template type can be found <a href="templatedtd.html">here</a> (in pretty
- * html format) or <a href="templatedtd.dtd">here</a> (in raw form). The XML is always verified and the first line
+ * The DTD for the XML description of a template type can be found <a href="template.html">here</a> (in pretty
```

Comment:
I don't know that `templatedtd` isn't correct, I think it should stay.

Reply:
I think the intent is to refer to the file named `template.dtd` in the same folder, so that change is good. I couldn't find and `templated*.html` so that reference should be left alone.

Reply:
I'm happy to drop it. When I was creating this commit, I left it in partially to talk about it. I can't find anything that explains what in the world this thing is. There's no code history to speak of, and no obvious references.

The only instance of the filename per google is this file: https://www.google.com/search?q=%22templatedtd.html%22

Is there anyone who might know what this is talking about? Offhand, it feels like two files were lost in open-sourcing: `templatedtd.html` and `oldtemplate.html`:
https://github.com/eclipse/openj9/blob/3ddd56b705757e978be121a87c932e3d37c95206/debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/tdump/zebedee/util/Template.java#L42-L45
https://github.com/eclipse/openj9/blob/3ddd56b705757e978be121a87c932e3d37c95206/debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/tdump/zebedee/util/template.dtd#L14-L16
(That the author is the anonymous `I` doesn't help matters.)

And note, while there are links to `templatedtd.dtd` and `templatedtd.html`, the only file in this directory is `template.dtd`.

I think the right thing to do is for me to create an issue for this item because it's really a significant bug as opposed to a simplistic typo.

Reply:
@pshipton, @keithc-ca: I've split this out as #5927